src/main/groovy/com/craigburke/gradle/client/registry/core/AbstractRegistry.groovy

```diff
@@ -151,7 +151,7 @@ abstract class AbstractRegistry implements Registry {
                 infoFile.parentFile.mkdirs()
                 infoFile.text = new JsonBuilder(dependency.info).toPrettyString()
             }
-
+            dependency.info = dependency.info ?: [:]
```

Comment:
If `depencency.info` is null, then https://github.com/craigburke/client-dependencies-gradle/blob/master/src/main/groovy/com/craigburke/gradle/client/registry/bower/GithubResolver.groovy#L88 and https://github.com/craigburke/client-dependencies-gradle/blob/master/src/main/groovy/com/craigburke/gradle/client/registry/bower/GithubResolver.groovy#L92 will throw NPE's